runtime/compiler/net/MessageTypes.hpp

```diff
@@ -135,6 +135,7 @@ enum MessageType : uint16_t
    VM_isPrimitiveArray,
    VM_getAllocationSize,
    VM_getObjectClass,
+   VM_getObjectClassAt,
```

Comment:
Could you please update message numbers that come after this message?

Reply:
Updated